<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>210</width>
     <height>80</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>로고</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>168</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>GotoWork</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_nowdate">
      <property name="text">
       <string>오늘의 날짜</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_nowtime">
      <property name="text">
       <string>시간</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_attendance">
      <property name="text">
       <string>출근</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_leavework">
      <property name="text">
       <string>퇴근</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
